staging/src/k8s.io/api/core/v1/types.go

```diff
@@ -3085,6 +3091,19 @@ type PodDNSConfigOption struct {
 	Value *string `json:"value,omitempty" protobuf:"bytes,2,opt,name=value"`
 }
 
+// An EphemeralContainer is a container which runs temporarily in a pod for human-initiated actions
+// such as troubleshooting. This is an alpha feature enabled by the EphemeralContainers feature flag.
+type EphemeralContainer struct {
+	Container `json:",inline" protobuf:"bytes,1,opt,name=container"`
```

Comment:
Hrm, this should be inlined by protobuf when you define it as an inline container.